src/PrestaShopBundle/Resources/config/services/adapter/supplier.yml

```diff
@@ -13,6 +13,7 @@ services:
 
   prestashop.adapter.supplier.supplier_order_validator:
     class: 'PrestaShop\PrestaShop\Adapter\Supplier\SupplierOrderValidator'
+    deprecated: ~
```

Comment:
Could you use the recommended message for deprecations : https://devdocs.prestashop.com/1.7/development/coding-standards/#deprecations ?

Reply:
Fixed